docker-images/kafka-mirror-maker/scripts/kafka_mirror_maker_run.sh

```diff
@@ -61,7 +61,7 @@ if [ -z "$KAFKA_HEAP_OPTS" -a -n "${DYNAMIC_HEAP_FRACTION}" ]; then
 fi
 
 if [ -n "$KAFKA_MIRRORMAKER_WHITELIST" ]; then
-    whitelist="--whitelist ${KAFKA_MIRRORMAKER_WHITELIST}"
+    whitelist="--whitelist \""${KAFKA_MIRRORMAKER_WHITELIST}"\""
```

Comment:
Does this actually avoid glob expansion when the whitelist is, or contains, `*`?  I think it should use quoted single quotes, like this

```
whitelist="--whitelist \'"${KAFKA_MIRRORMAKER_WHITELIST}"\'"
```

But I might be wrong.

Reply:
I did some tests to find the proper way. Just to convince you :P
```
[sknot@sknot strimzi]$ export REGEX=".*"
[sknot@sknot strimzi]$ export TEST="\'"${REGEX}"\'"
[sknot@sknot strimzi]$ echo $TEST
\'.*\'
[sknot@sknot strimzi]$ export TEST="\""${REGEX}"\""
[sknot@sknot strimzi]$ echo $TEST
".*"
```

Reply:
Yep, you're right: https://unix.stackexchange.com/questions/67757/wildcards-inside-quotes